lib/blockscore/person.rb

```diff
@@ -10,7 +10,7 @@ class Person < Base
 
     attr_reader :question_sets
 
-    def initialize(options = {})
+    def initialize(*)
```

Comment:
I usually call `super` early on. I believe if you're adhering to [LSP](https://en.wikipedia.org/wiki/Liskov_substitution_principle), the subclass will generally have compatible instance state with it's ancestor class. If there are differences or additions I will perform them after `super`.

It *does* depend on the method though. I follow the above advice for `#initialize` and `.new`. Some methods may require you to step in before `super`, but I find those relatively rare.